components/malloc_size_of/Cargo.toml

```diff
@@ -34,4 +34,4 @@ url = { workspace = true }
 uuid = { workspace = true }
 webrender_api = { workspace = true }
 wr_malloc_size_of = { workspace = true }
-xml5ever = { workspace = true }
+markup5ever = { workspace = true }
```

Comment:
Please keep this in alphabetical order!

Reply:
Dammit. I did it for the main `Cargo.toml` but missed the `malloc_size_of` one